docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -15,40 +15,271 @@ keywords:
+```tsx title="src/components/Toolbar.tsx"
+import { usePluginComponentExtensions } from '@grafana/runtime';
+
+export const Toolbar = () => {
+  // This only returns type="component" extensions
+  // Heads up! We don't specify a context object below, we pass in the contextual information as a prop to the component later.
+  const { extensions } = usePluginComponentExtensions({ extensionPointId: '<extension-point-id>' });
+
+  return (
+    <div>
+      <div className="title">Title</div>
+      <div className="extensions">
+        {/* Loop through the available extensions */}
+        {extensions.map((extension) => {
+          const Component = extension.component as React.ComponentType<{
+            version: string;
+          }>;
+
+          // Render extension component and pass contextual information (version)
+          return (
+            <div key={extension.id}>
+              <Component version="1.0.0" />
+            </div>
+          );
+        })}
+      </div>
+    </div>
+  );
+};
+```
+
+### `getPluginExtensions()` - _deprecated_
+
+The `getPluginExtensions` method takes an object consisting of the `extensionPointId`, which must begin `plugins/<PLUGIN_ID>`, and any contextual information that you want to provide. The `getPluginLinkExtensions` method returns a list of extension links that your program can then loop over.
+
+:::note
+
+This function only returns the state of the extensions registry (the extensions registered by plugins) at a given time. If there are extensions registered by plugins after that point in time, you won't receive them. <br />
+**We strongly suggest to use the reactive [`usePluginExtensions()`](#usepluginextensions) hook instead wherever possible.**
+
+:::
+
+#### Syntax
+
+```tsx
+getPluginExtensions(options);
+getPluginLinkExtensions(options); // Only returns extensions that have type `type="link"`
+getPluginComponentExtensions(options); // Only returns extensions that have type `type="component"`
+```
+
+#### Parameters
+
+##### `options.extensionPointId` - _string_
+
+The unique identifier of your extension point. It must begin with `plugins/<PLUGIN_ID>`, for example: `plugins/myorg-super-app`.
+
+##### `options?.context` - _object (Optional)_
+
+An arbitrary object, that contains information related to your extension point which you would like to share with the extensions, for example: `{ baseUrl: '/foo/bar' }`. This parameter is not available for component extensions, for those you can pass contextual information using the component props. **Note:** the provided context object always gets frozen (turned immutable) before being shared with the extensions.
+
+##### `options?.limitPerPlugin` - _number (Optional)_
```

Comment:
```suggestion
##### The `options?.limitPerPlugin` method - _number (Optional)_
```

Reply:
I guess this suggestion is maybe a typo? (This is one of the option properties). I will resolve it, please reopen if you have objections!